bin/raar-record-handler.sh

```diff
@@ -86,8 +102,41 @@ do
 
     echo "Archiving ${eventFileName}"
 
-    tmpPath="${destDir}/.${eventFileName}.tmp"
-    destPath="${destDir}/${eventFileName}"
+    # Get the duration of the recording in seconds, with microsecond accuracy
+    # such as "3599.998979"
+    ffprobeOutput="$( LC_ALL=C ffprobe -i "${sourcePath}" \
+                                       -show_entries format="duration" \
+                                       -print_format csv="print_section=0" \
+                                       -v quiet )"
+
+    if [ $? -eq 0 ]; then
+        # Round to integer value in seconds
+        # 3599.998979 => 3600
+        duration=$( LC_ALL=C printf '%.0f' "$ffprobeOutput" )
+
+        echo "Duration of recording is ${duration} seconds (${ffprobeOutput})"
+    else
+        echo "Unable to determine duration of recording" >&2
+        duration="0"
```

Comment:
If nobody actively looks at stderr, theses errors will remain unnoticed. The filenames will contain a duration of zero seconds and therefore will not be imported by raar. So it is important to get a notification in Zabbix if this fails. Can zabbix_sender be used for this?
When a notification is received, an administrator could manually check the length of the recording and adjust the filename accordingly, so it would be imported in the next run.

Reply:
fixed and force-pushed - thanks @codez 

The recording file will now be skipped if the duration couldn't be determined. E.g. `ffprobe` exited with a non-zero exit status or the output doesn't look like an integer or float.

Skipping the recording file, won't trigger the existing `zabbix_sender` command which in turn won't send a successful recording notification to the monitoring system. Thus, the monitoring system will trigger an alarm, as it's missing the last successful recording.

The failed recording file, is still available within the rotter recording directory for a later analysis.

Reply:
👍